consensus/cometbft/service/finalize_block.go

```diff
@@ -23,11 +23,19 @@ package cometbft
 import (
 	"context"
 	"fmt"
+	"time"
 
 	cmtabci "github.com/cometbft/cometbft/abci/types"
 	"github.com/sourcegraph/conc/iter"
 )
 
+// Delay to use before the upgrade to SBT.
+const constBlockDelay = 1 * time.Second
+
+// Height to enable SBT. Changes will be applied in the next block after this
+// (10_000_001).
+const sbtEnableHeight = 10_000_000
```

Comment:
Have different heights for consensus parameters updates and sbtEnableHeight
```suggestion
const sbtEnableHeight = 10_000_000
const sbtConsensusParamUpdateHeight = 10_00_0
```